patterns/1-initial/overcoming-project-management-time-pressures.md

```diff
@@ -20,10 +20,10 @@ Customer deadlines and delivery commitments for feature content do not change.
 - Project management concern that engagement will lead to missed deadlines
 - Project management concern that Code contribution and/or mentoring may lead to dilution of subject matter expert time spent on their own projects (other teams may require their time).
 
-+ Elimination of rewriting common code saves time (write once, use many times)
-+ Crowd-based testing and debugging saves time (and improves quality)
-+ The collaboration and synergy of inner sourcing can generate new, innovative features  .
-+ Test Driven Development could mitigate some of the above forces
+- Elimination of rewriting common code saves time (write once, use many times)
```

Comment:
Visual change since this collapses them down to a single unordered list. Not sure what the original intention was